.circleci/config.yml

```diff
@@ -57,10 +57,10 @@ jobs:
             pip install --progress-bar off .[checking]
 
       - run:
-          name: autopep8
+          name: black
           command: |
             . venv/bin/activate
-            autopep8 . -r --diff --exit-code
+            black . --line-length 99 --check --exclude="docs"
```

Comment:
Fine. 
FYI
If we want to use a configration file, we can use `pyproject.toml` for black.

Reply:
A drop by, leaving a reference https://github.com/optuna/optuna/pull/1020#discussion_r392775740